packages/ic-response-verification/tests/v2_response_verification_certification_scenarios.rs

```diff
@@ -346,6 +374,18 @@ mod fixtures {
         }
     }
 
+    #[fixture]
+    pub fn index_js_certification() -> HttpCertificationTreeEntry<'static> {
+        HttpCertificationTreeEntry {
```

Comment:
should we make it consistent with how this is being used above? `HttpCertificationTreeEntry::new`

Reply:
The issue here is on line 381. Since this is being returned from a function, we need to take the `HttpCertification` struct by value or Rust is complaining that the value doesn't live long enough 👴